src/hotspot/share/classfile/vmIntrinsics.cpp

```diff
@@ -316,6 +314,10 @@ bool vmIntrinsics::disabled_by_jvm_flags(vmIntrinsics::ID id) {
   case vmIntrinsics::_fmaF:
     if (!InlineMathNatives || !UseFMA) return true;
     break;
+  case vmIntrinsics::_dtanh:
+  case vmIntrinsics::_dcbrt:
+    if (!InlineMathNatives) return true;
```

Comment:
Ah, looks like you missed !InlineIntrinsic check here:
if (!InlineMathNatives || !InlineIntrinsics) return true;

Reply:
Yes, I was thinking to fall through to the cases below, but I missed the break statement. It's clearer with the extra check anyway, so I added it in now.